src/core/operations/LengthValueDecoder.mjs

```diff
@@ -0,0 +1,102 @@
+/**
+ * @author gchq77703 []
+ * @copyright Crown Copyright 2018
+ * @license Apache-2.0
+ */
+
+import Operation from "../Operation";
+import LengthValue from "../lib/LengthValue";
+
+/**
+ * From Length Value operation
+ */
+class FromLengthValue extends Operation {
+
+    /**
+     * FromLengthValue constructor
+     */
+    constructor() {
+        super();
+
+        this.name = "From Length Value";
+        this.module = "Default";
+        this.description = "Converts a Length-Value (LV) encoded string into a JSON object.  Can optionally include a <code>Key</code> / <code>Type</code> entry.";
+        this.infoURL = "";
+        this.inputType = "byteArray";
+        this.outputType = "JSON";
+        this.args = [
+            {
+                name: "Bytes in Key Value",
+                type: "populateOption",
```

Comment:
This is the wrong argument type to use. Take a look here for a description of each of the ingredient types: https://github.com/gchq/CyberChef/wiki/Adding-a-new-operation#data-types